tests/active_features/site-team.feature

```diff
@@ -18,20 +18,20 @@ Feature: Managing a site's team
 
   @vcr site_team_change-role
   Scenario: Changing a team member's role
-    When I run "terminus site:team:role --site=[[test_site_name]] --member=[[other_user]] --role=admin"
+    When I run "terminus site:team:role [[test_site_name]] --member=[[other_user]] --role=admin"
     Then I should get one of the following: "This site does not have its change-management option enabled., Changed a user role"
 
   @vcr site_team_list
   Scenario: Listing team members
-    When I run "terminus site:team:list --site=[[test_site_name]]"
+    When I run "terminus site:team:list [[test_site_name]]"
```

Comment:
@ari-gold Is this an ok change? Site is not optional for the command so it should be a parameter, IMHO.